src/main/antlr4/FIRRTL.g4

```diff
@@ -87,6 +87,16 @@ field
   : 'flip'? id ':' type
   ;
 
+extModuleName
```

Comment:
The FIRRTL IR definition call this extname but serializes as 'name ='. In the internal IR name is actually the overall defname. The inconsistency is undesirable.